### ISSUE TITLE
Problema create new feature with subfolder

### ISSUE THREAD

**Issue by ferndem** [jakobehn/GitFlow.VS#94]
I have a problem when create a new feature with subfolder, ex: <folder1>/<feature-name>

When I run command from prompt 
 _**git flow feature start <folder1>/<feature-name>**_

the correct branch is created.
If i create feature from plugin it create only <feature-name> branch

**Comment by jakobehn** [comment]
Really late here, but I can't reproduce this problem. When creating a new feature called feature/new-feature, it gets created correctly:

![image](https://user-images.githubusercontent.com/847244/148206533-6eef90f3-34d2-4b4b-8deb-83b635c0bf2b.png)

**Comment by ferndem** [comment]
The problem is when using the GitFlow plugin and selecting "Start Feature"

![image](https://user-images.githubusercontent.com/39851927/148209497-812c6f26-cd72-449e-9caf-07eda19e1e02.png)

**Comment by jakobehn** [comment]
Yes, I know. That's what I did in the example above. the branch get created correctly, however the extension only shows the end name of the branch, maybe that is what you are referring to?

![image](https://user-images.githubusercontent.com/847244/148213518-f45b3bae-a205-44e9-a321-d836250f827f.png)

**Comment by ferndem** [comment]
The problem is if I insert a subfolder that is:
fd/new-feature

![image](https://user-images.githubusercontent.com/39851927/148214491-084ac623-89f8-4a79-af94-3b58b61989f6.png)

create just "new-feature" and not "fd/new-feature"

![image](https://user-images.githubusercontent.com/39851927/148214652-f64cf965-1fa8-4366-8511-447ff789a2f3.png)

**Comment by jakobehn** [comment]
It actually creates the correct branch (fd/new-feature in your case), you can see that in the list of branches. 
However the extension only shows the name of the branch. so that is a bug. I'll mark this as a bug and will fix it
